jenkins/requirements.txt

```diff
@@ -1 +0,0 @@
-requests==2.5.3
```

Comment:
``pip`` is terrible at multiple things have the same dependency listed so I factored it up to the top level (apparently this *just* got fixed)